simexpal/launch/queue.py

```diff
@@ -27,17 +27,19 @@ def submit(self, cfg, run):
 
 		if not common.lock_run(run):
 			return
-		common.create_run_file(run)
 
 		specfd, specfile = tempfile.mkstemp(prefix='', suffix='-spec.yml',
 											dir=os.path.join(cfg.basedir, 'aux/_queue'))
 
+		queue_jobid = util.extract_file_prefix_from_path(specfile, '-spec')
```

Comment:
Is `extract_file_prefix_from_path` also returning a valid `queue_jobid`? If not, we should add proper error handling.

Reply:
Yes, ``extract_file_prefix_from_path()`` reads the queue_jobid from the file created by ``tempfile.mkstemp()`` in the lines before, so it will always be a valid queue_jobid